internal/server/data/organization.go

```diff
@@ -50,6 +50,14 @@ func CreateOrganization(tx WriteTxn, org *models.Organization) error {
 		return fmt.Errorf("failed to create infra provider: %w", err)
 	}
 
+	// social login is enabled by default when configured
+	if googleProvider != nil {
+		googleProvider.OrganizationMember = models.OrganizationMember{OrganizationID: org.ID}
```

Comment:
As a safeguard maybe we should also set `googleProvider.Managed = true` here, to prevent accidentally saving it as not-managed.

 I think that would also require the test for `CreateUpdated` be updated to check that `Managed = true` was set, which would be nice.

Reply:
This managed field went away with one Google provider being shared for all orgs